elements/pl-workspace/pl-workspace.py

```diff
@@ -1,6 +1,12 @@
 import chevron
 
 
+def add_format_error(data, error_string):
+    if '_files' not in data['format_errors']:
+        data['format_errors']['_files'] = []
+    data['format_errors']['_files'].append(error_string)
+
+
 def render(element_html, data):
```

Comment:
As @nicknytko pointed out, we should render differently depending on which panel we are in.